pkg/spec/spec.go

```diff
@@ -394,43 +398,6 @@ func (config *CreateConfig) createConfigToOCISpec(runtime *libpod.Runtime, userM
 		}
 	}
 
-	// Make sure that the bind mounts keep options like nosuid, noexec, nodev.
-	mounts, err := pmount.GetMounts()
```

Comment:
@rhatdan I think you originally wrote this, so you might want to take a look. I made adding `nosuid`/`noexec`/`nodev` unconditional (unless the user gives a `suid`/`exec`/`dev` to counter), not based on this pmounts call here. I think this is fine?

Reply:
nosuid and nodev maybe.  Noexec could be an issue.  Does this affect volume mounts?  IE
If I volume mount in /usr/bin.  I probably don't want it marked as noexec.

Reply:
Fair point. I'll poke at it more...